js/client-mainmenu.js

```diff
@@ -1224,16 +1224,42 @@
 	});
 
 	var FormatPopup = this.FormatPopup = this.Popup.extend({
+		events: {
+			'keyup input': 'updateSearch',
+			'click details': 'updateOpen',
+		},
 		initialize: function (data) {
-			var curFormat = data.format;
+			this.data = data;
+			if (!this.open) {
+				// todo: maybe make this configurable? not sure since it will cache what users toggle.
+				// avoiding that decision for now because it requires either an ugly hack
+				// or an overhaul of BattleFormats.
+				this.open = Storage.prefs('openformats') || {
+					"S/V Singles": true, "S/V Doubles": true, "National Dex": true,
+					"OM of the Month": true, "Randomized Format Spotlight": true, "RoA Spotlight": true,
```

Comment:
```suggestion
					"S/V Singles": true, "S/V Doubles": true, "National Dex": true, "OM of the Month": true,
					"Other Metagames": true, "Randomized Format Spotlight": true, "RoA Spotlight": true,
```
This should non negotiably be shown.

Reply:
Oh, you left that off the list I think. mb